langdetect/detector_factory.py

```diff
@@ -146,10 +149,21 @@ def detect(text, supported_languages=None):
 
     return detector.detect()
 
+def is_chinese(predicted_language):
+    return predicted_language in ["zh", "zh-tw"]
+
+def predict_simplified_traditional(text):
+    predicted_language = "zh" if mafan.text.is_simplified(text) else "zh-tw"
+
+    return [Language(lang=predicted_language, prob=0.999999999)]
```

Comment:
feels a bit weird to be overriding the probability of the initial language detection model - or does it make more sense this way?

Reply:
Actually, I think we should keep the original probability(s). Just because a Chinese language is the top predicted language, doesn't mean the model should be confident, right?

Reply:
That is right! Good suggestion, will make the change now.